package.json

```diff
@@ -3,19 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.3",
```

Comment:
**You guys did great job! I like the implemetation you did in this project.** 

- [ ] However, for this project students should write unit tests with React Testing Library and Jest (for unit tests). But in your project. I was not able to find any test releted to this project. Kindly add unit tests and resubmit again. Thank you! :)